x-pack/plugins/apm/public/components/alerting/rule_types/register_apm_rule_types.ts

```diff
@@ -16,13 +16,9 @@ import {
 import { ApmRuleType } from '../../../../common/rules/apm_rule_types';
 import {
   anomalyMessage,
-  anomalyRecoveryMessage,
```

Comment:
Did you leave the messages on purpose ? https://github.com/elastic/kibana/blob/e2ae42f66f03dbe9e37c7821eb50012718b97d8e/x-pack/plugins/apm/common/rules/default_action_message.ts#L132

if the recovery messages are not used anywhere can we clean it? We can always bring it back

Reply:
Yes, I kept them because we will bring them back after https://github.com/elastic/kibana/issues/158183 is done.

Reply:
Here is a ticket for it: https://github.com/elastic/kibana/issues/160969

Reply:
thanks 🙏